src/Illuminate/Support/Str.php

```diff
@@ -53,6 +53,16 @@ public static function of($string)
         return new Stringable($string);
     }
 
+    /**
+     * Get a new stringable object.
+     *
+     * @return \Illuminate\Support\Stringable
+     */
+    public static function builder()
```

Comment:
Why not add a `make($string = '')` as an alias to `of($string)`?

or even change the already existing `of(...)` function to `of($string = '')` (i'd be in favor for renaming but that causes breaking changes).